### ISSUE TITLE
opensc 0.16.0 ships broken bash completion scripts

### ISSUE THREAD

**Issue by acdha** [Homebrew/homebrew-core#5338]
opensc 0.16.0 generates a bunch of bash completion scripts which are malformed, which means that if you `brew install opensc` the next time you start a shell you'll get this:

```
-bash: /usr/local/etc/bash_completion.d/cardos-tool: line 18: syntax error near unexpected token `)'
-bash: /usr/local/etc/bash_completion.d/cardos-tool: line 18: `             )'
… repeated for dozens of scripts …
```

This was fixed in https://github.com/OpenSC/OpenSC/issues/793 so `brew install opensc --HEAD` works correctly but it's not clear to me from https://github.com/OpenSC/OpenSC/issues/813 whether that update is being prioritized.

**Comment by dunn** [comment]
If it's a single commit, we can backport it to the current stable version.  Take a look at https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md if you're interested in opening a PR!

**Comment by acdha** [comment]
That link is stale – it should be https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md – but to answer the question, it does appear to be a single commit. I'll look into the current patching process later – it's been awhile since I've sent a PR.

**Comment by anthonysomerset** [comment]
So the problem as i see it... OpenSC have fixed it since the 0.16.0 release but have not made a new release since then - brew can't fix this unless we explicitly build from HEAD which could be a bad thing... my suggestion would be to ask Upstream (OpenSC) to tag a new release at which point its trivial to as i have done in https://github.com/OpenSC/OpenSC/issues/813

**Comment by fxcoudert** [comment]
If upstream does not release an update, can we get a pull request with `patch` using `https://github.com/OpenSC/OpenSC/commit/2746a7f430cb02fbd21cf9c2f0819124e6adca76.diff`
